modules/campaignion_petition/campaignion_petition_templates/campaignion_petition_templates.info

```diff
@@ -12,6 +13,6 @@ dependencies[] = webform
 __drupal_alter_by_ref[][node][petition] = petition
 features[ctools][] = strongarm:strongarm:1
 features[features_api][] = api:2
+features[uuid_node][] = 0e6c7ce6-5701-4c17-b306-be7eba55dfe8
```

Comment:
If the uuids changed we must also change the default template in `campaignion_donation_type_campaignion_action_info()`.